src/EventHandler.hs

```diff
@@ -4,28 +4,25 @@ module EventHandler
   ( handleKeys
   ) where
 
-  import GameBoard
-  import Physics
+import GameBoard
+import Physics
 
-  import Graphics.Gloss.Interface.Pure.Game
-  import System.Exit
+import Graphics.Gloss.Interface.Pure.Game
+import System.Exit
 
-<<<<<<< ef207458608785b7998e8f3d8c39008da8a43327
-=======
+-- | Pure responding to key events.
+handleKeys :: Event     -- ^ keyEvent
+           -> Game      -- ^ Initial game state
+           -> Game      -- ^ Game updated
+-- For an 'Left' or 'Right' keypress, move verticaly player1 paddle
```

Comment:
an 'Left' -> a left. We are not referring to the `Left` and `Right` constructors of `Either` !

verticaly -> vertically.